Controller/TransactionsController.php

```diff
@@ -38,7 +38,13 @@ public function exportAction(Request $request, $objectId)
         if (empty($objectId)) {
             return $this->accessDenied();
         }
+
+        if (!$this->get('mautic.security')->isAdmin() && $this->get('mautic.security')->isGranted('contactclient:export:disable')) {
```

Comment:
This feels backwards. A comment would help us understand the thought behind it.

If not an admin, and they have the permission "contactclient:export:disable"?

So this is an anti-permission? That feels backwards...

Reply:
LOL..I will add the comment...
Lead:Export:disabled - when true, export is disabled
Admins return true for all permissions
Therefore, we only heed the Lead:Export:Disable when NOT Admin

Reply:
I dont know why core was handled as an anti-pattern vs just creating a migration that adds the bit for each role (how many possible roles could there be?) But, its what they did...